### PR TITLE
Syndicate Ninja™ box set

### PR DESCRIPTION
**Pull request by yackemflam**
:cl: Yackemflam
add: You can now have a chance to be an official Syndicate Ninja™!
add: Just buy a syndicate box for a chance to get your very own official Syndicate Ninja™ set!
add: It contains your amazingly sharp katana, throwing stars with belt, ninja training in a implanter, and an ninja badass armor set to fight the Nanotrasen scum!
/:cl:
image
image

### CODE REVIEW

**Comment by PKPenguin321** [comment]
>sneaking in assistant clumsiness

:-1: x :100:

**Comment by PKPenguin321** [comment]
wait i lied you removed it
you still have remnants of it though

File: code/game/gamemodes/traitor/traitor.dm [hunk 1]
```diff
@@ -309,6 +309,11 @@
 			traitor_mob << "Your training has allowed you to overcome your clownish nature, allowing you to wield weapons without harming yourself."
 			traitor_mob.dna.remove_mutation(CLOWNMUT)
 
+	if (traitor_mob.mind)
+		if (traitor_mob.mind.assigned_role == "Assistant")
+			traitor_mob << "Your training has allowed you to overcome your graysuit nature, allowing you to wield weapons without harming yourself."
+			traitor_mob.dna.remove_mutation(CLOWNMUT)
+
```
Comment (PKPenguin321) [inline code/game/gamemodes/traitor/traitor.dm]: delet all this

**Comment by yackemflam** [comment]
@PKPenguin321, You can check the code, there is no additional coding for assistant clumsiness.

**Comment by PKPenguin321** [comment]
the armor values on this are wiener and it's basically an concealable esword with some throwing stars which kinda suck

if i got this i would be pretty dissapointed tbh. it's a fun idea but not nearly good enough for 20 TCs

**Comment by yackemflam** [comment]
30 force unconcealable esword <8 tc
adrenals 8 tc
6 ninja stars >5 tc
Some armor
Seems about right for a 20 tc box set

Remember, the 'murder' set is EXACTLY 20 tc.

**Comment by PKPenguin321** [comment]
\>walk out of maint in your shitty armor that barely blocks anything and reveals your antag status
\>first person you see calls over radio that you're a traitor
\>disarms your sword that you cant put away or hide
\>sec comes and stuns you
\>you adrenal back up and throw a star at one of them which does like a little damage i guess
\>eventually get stunned again and beaten to death

if i made a ninja set i would go with more stealthy stuff rather than items that instantly give you away that you can't conceal
like maybe a DNA injector with chameleon

**Comment by yackemflam** [comment]
Did I mention that since this weapon has 30 force. it has a 30% chance to KO you and a 40% chance to knock you down.
AND it still inherits it's 50% chance to block ALL forms of attacks.

**Comment by PKPenguin321** [comment]
still sucks

**Comment by yackemflam** [comment]
Well, so does most of the boxes anyhow.

**Comment by optimumtact** [comment]
Then don't pick it @PKPenguin321

**Comment by yackemflam** [comment]
You CAN'T not choose it if you buy box roulette. it's rng for a reason.

**Comment by Archie700** [comment]
>exactly 20 tc

You should probably add a cham projector. Boxes have items more than 20 TC when added up.

**Comment by Niknakflak** [comment]
That name "mallninja" and "mninja" are ew

File: code/game/objects/items/weapons/storage/uplink_kits.dm [hunk 2]
```diff
@@ -101,6 +101,24 @@
...
+			new /obj/item/weapon/throwing_star(src)
+			new /obj/item/weapon/storage/belt/military/assault(src)
+			new /obj/item/clothing/shoes/sandal/mninja(src)
+			new /obj/item/clothing/head/fedora/mninja(src)
+			new /obj/item/clothing/suit/jacket/leather/overcoat/mninja(src)
+			new /obj/item/clothing/under/pants/black(src)
+			new /obj/item/clothing/gloves/fingerless(src)
+			new /obj/item/clothing/glasses/sunglasses(src)
+			new /obj/item/clothing/mask/bandana/skull(src)
+			return
```
Comment (Jalleo) [inline code/game/objects/items/weapons/storage/uplink_kits.dm]: Requesting a agebt ID card and a emag for quick entries and disguises (aka no AI track)

File: code/modules/clothing/shoes/miscellaneous.dm [hunk 3]
```diff
@@ -41,6 +41,11 @@
 	desc = "A pair of magic black shoes."
 	name = "magic shoes"
 	icon_state = "black"
+	
+/obj/item/clothing/shoes/sandal/mninja
+	name = "Syndicate Ninja Sandals"
+	desc = "An official Syndicate Ninja Sandals. They have thin metal platings on the bottom. You feel that you could block melee attacks with these."
+	armor = list(melee = 5)
```
Comment (Jalleo) [inline code/modules/clothing/shoes/miscellaneous.dm]: Does shoes actually work in armor protection?

**Comment by Jalleo** [comment]
I think this is alright but people will probably not use the armor in this and just take the rest of it.

**Comment by anconfuzedrock** [comment]
I fucking hate bundles they need to be removed/reworked

**Comment by yackemflam** [comment]
@Archie700 The murder box comes with 20 tc worth of items
esword - 8 tc
emag - 6 tc
thermals - 4 tc
no slips - 2 tc

File: code/modules/clothing/shoes/miscellaneous.dm (hunk 3 above)
Comment (yackemflam) [inline code/modules/clothing/shoes/miscellaneous.dm]: DS have armored boots

**Comment by yackemflam** [comment]
Edited the armor values

**Comment by Archie700** [comment]
This is less ninja and more euphoric.

Should name it the "edgelord" or "euphoric" set

**Comment by yackemflam** [comment]
@Archie700 did you even read the box set name? And the clothes? It's a ninja.

**Comment by TechnoAlchemisto** [comment]
thats not what a ninja looks like

**Comment by yackemflam** [comment]
I know, that's the point.

**Comment by TechnoAlchemisto** [comment]
looks bad

**Comment by yackemflam** [comment]
That is the point.

**Comment by ShadowDeath6** [comment]
>add: It contains your amazingly sharp katana, throwing stars with belt, ninja training in a implanter, and **an ninja** badass armor set to fight the Nanotrasen scum!
a ninja*

why is this even ninja if there's nothing ninja-esque other than the weapons?

**Comment by yackemflam** [comment]
That's the fucking point.
These are Chinese fucking knockoffs

**Comment by PKPenguin321** [comment]
its also a shitty traitor bundle

**Comment by yackemflam** [comment]
I gave it armor what more do you want?

**Comment by optimumtact** [comment]
Welcome to the balance seesaw @yackemflam 

some advice, you can safely ignore 99% of people

**Comment by yackemflam** [comment]
It's not even a seesaw at this point.

**Comment by PKPenguin321** [comment]
yeah it's just a shitty set

**Comment by yackemflam** [comment]
What do YOU suggest then?

**Comment by PKPenguin321** [comment]
not adding it

it takes the previously usually decent traitor bundles and adds a really shitty dud to the mix that has no point of being there

**Comment by Cheridan** [comment]
@yackemflam It's a bad idea to add ironically bad joke items because people will eventually want to make them good (see dualsabers)

**Comment by yackemflam** [comment]
>30 force weapon with 50% block chance of both melee and projectiles
>6 shurikens, they fucking hurt if you know how to aim
>a belt to hold them
>Some armor
>Adrenals to boot
This is pretty on par with the other box sets too.

**Comment by kilkun** [comment]
we already have ninjas in the game.
:-1:

**Comment by Cheridan** [comment]
downthumbs

**Comment by kevinz000** [comment]
rip

**Pull request closed without merging**